packages/hoprd/src/api/v3.ts

```diff
@@ -182,7 +192,7 @@ export async function setupRestApi(
         }
         return true
       },
-      address: (input) => {
+      ethereumaddress: (input) => {
```

Comment:
Shouldn't it be lower camel-case ?

Reply:
We have a normalization in place which makes all keys lower case.